src/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/gru_sequence.hpp

```diff
@@ -43,3 +45,36 @@ class GRUSequenceTest : public testing::WithParamInterface<GRUSequenceParams>,
 };
 
 }  // namespace LayerTestsDefinitions
+
+namespace ov {
+namespace test {
+
+using GRUSequenceParams = typename std::tuple<
+        ov::test::utils::SequenceTestsMode,       // pure Sequence or TensorIterator
+        size_t,                                   // seq_lengths
+        size_t,                                   // batch
+        size_t,                                   // hidden size
+        // todo: fix. input size hardcoded to 10 due to limitation (10 args) of gtests Combine() func.
+        //size_t,                                 // input size
+        std::vector<std::string>,                 // activations
+        float,                                    // clip
+        bool,                                     // linear_before_reset
+        ov::op::RecurrentSequenceDirection,       // direction
+        ov::test::utils::InputLayerType,          // WRB input type (Constant or Parameter)
+        ov::element::Type,                        // Network precision
+        std::string>;                             // Device name
+
+class GRUSequenceTestNew : public testing::WithParamInterface<GRUSequenceParams>,
```

Comment:
I think you can avoid New postfix, new classes are located in new namespace

Reply:
It is OK from C++ perspective. But not from Gtest.
Namespace doesn't take part in Gtest test name mangling process. So we have to change these names.

I can just use original name for new class GRUSequenceTest and rename old class GRUSequenceTestLegacy

Reply:
I cannot change legacy test names because these names are still used in openvino_contrib repo

Reply:
See my comment to `src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/broadcast.cpp`